docs/sources/mimir/configure/configure-otel-collector.md

```diff
@@ -107,6 +107,41 @@ service:
       exporters: [..., prometheusremotewrite]
 ```
 
+## Work with default OpenTelemetry labels
+
+OpenTelemetry metrics use resource attributes to describe the set of characteristics associated with a given resource, or entity producing telemetry data. For example, a host resource might have multiple attributes, including an ID, an image, and a type.
```

Comment:
The way I understand the paragraph, entity should be a synonym for resource, let's make it clear through a strategic comma:
```suggestion
OpenTelemetry metrics use resource attributes to describe the set of characteristics associated with a given resource, or entity, producing telemetry data. For example, a host resource might have multiple attributes, including an ID, an image, and a type.
```